infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -126,8 +126,8 @@
      * Handler for this server.
      */
     @Getter
-    private CorfuMsgHandler handler = new CorfuMsgHandler()
-            .generateHandlers(MethodHandles.lookup(), this);
+    private final CorfuMsgHandler handler =
+        CorfuMsgHandler.generateHandler(MethodHandles.lookup(), this);
 
     private static final String metricsPrefix = "corfu.server.sequencer.";
     private static Counter counterTokenSum;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "counterTokenSum" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)